src/components/dropdown/index.js

```diff
@@ -129,7 +129,6 @@ const Dropdown = ({
         dropUp={dropUp}
         leftAlign={leftAlign}
         minWidth={dropdownWidth || "160px"}
-        ref={ref}
```

Comment:
For some reason, after updating React, the placement of this `ref` was no longer working. (To be honest, I'm not sure how it ever was working, since the trigger `Button` is outside of it and the `handleClickOutside` function should have been running every time you clicked the button, thus causing the dropdown to immediately hide.)